templates/definition/meter/marstek-venus.yaml

```diff
@@ -0,0 +1,166 @@
+template: marstek-venus
+products:
+  - brand: Marstek
+    description:
+      generic: Venus Battery Storage
+capabilities: ["battery-control"]
+requirements:
+  description:
+    en: >
+      Requires a Modbus TCP gateway (e.g., PUSR USR-DR134). Tested with firmware version V151.
```

Comment:
Sorry, discovering new things every time I look. Is that an rs485 to ethernet converter? In this case, does the Marstek have an rs485 connector but no ethernet? Then you should just use the modbus logic:

```yaml
  - name: modbus
    choice: ["rs485"]
    baudrate: 38400
    comset: 8E1
```

Reply:
Besseres Beispiel: https://github.com/evcc-io/evcc/blob/02d3a362b2d2a515b5ede6c66ab6ab3520f7cc86/templates/definition/charger/daikin-homehub.yaml#L4

Reply:
Ja und nein. Für Modbus wird der *physikalische* Anschluss konfiguriert, also rs485. Den Rest macht dann das inkludierte Template.